### ISSUE TITLE
multiline-blocks does not enforce noZeroLineText with noMultilineBlocks

### ISSUE THREAD

**Issue by danvk** [gajus/eslint-plugin-jsdoc#737]
## Expected behavior

The `multiline-blocks` rule normally disallows text on the zeroth line of a multiline block.

The code sample below produces the following error if you change the config to `{ noZeroLineText: true }`:

```
Should have no text on the "0th" line (after the `/**`). eslintjsdoc/multiline-blocks
```

## Actual behavior

If you set `noMultilineBlocks` and `minimumLengthForMultiline`, however,`noZeroLineText` is no longer enforced for longer comments. The example below is not flagged as an error.

## ESLint Config

```js
{
  noZeroLineText: true,
  noMultilineBlocks: true,
  minimumLengthForMultiline: 50
}
```

## ESLint sample

```js
/** This comment is bad
 * It should not have text on line zero.
 */
```

## Environment

- Node version: v12.16.3
- ESLint version v7.9.0
- `eslint-plugin-jsdoc` version: 34.6.1

**Comment by danvk** [comment]
Confirmed the fix, thanks so much!
